annual_dual_failure_probability.py

```
@@ -0,0 +1,111 @@
+HOURS_PER_YEAR = 8760
+
+
+# https://www.ti.com/lit/an/spraby3/spraby3.pdf?ts=1753039217595
+
+
+def main():
```

Ruff (0.12.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)